frontend/src/metabase-types/store/entities.ts

```diff
@@ -7,9 +7,16 @@ import {
   Table,
 } from "metabase-types/api";
 
+type EntityList<T> = Record<
+  any, // Usually a JSON stringified list query object or `null`
+  { list: T[]; metadata: { total?: number; limit?: number; offset?: number } }
+>;
+
 export interface EntitiesState {
   collections?: Record<CollectionId, Collection>;
   dataApps?: Record<DataAppId, DataApp>;
+  dataAppCollections?: Record<CollectionId, Collection>;
+  dataAppCollections_list?: EntityList<Collection>;
```

Comment:
@kulyk should we remove this and the EntityList type because it's not used? I'm not sure if we should use _list directly

Reply:
@ranquild fixed 👍